pkg/scheduler/framework/v1alpha1/types_test.go

```diff
@@ -622,15 +662,15 @@ func TestNodeInfoAddPod(t *testing.T) {
 			},
 		},
 		Requested: &Resource{
-			MilliCPU:         1300,
-			Memory:           1000,
+			MilliCPU:         2300,
+			Memory:           209716700, //1500 + 200MB in initContainers
```

Comment:
@fengzixu As discussed earlier, we can add a follow-up PR to adjust this behavior to check if sum(containers) is 0, instead of check individual container. The follow-up PR may only happen in master branch.

Reply:
Sure. I will create an issue to track it